src/commands/voice.js

```diff
@@ -61,7 +61,10 @@ module.exports = {
                     await DiscordMessages.sendVoiceMessage(interaction,
                         client.intlGet(interaction.guildId, 'commandsVoiceBotJoinedVoice'));
                     client.log(client.intlGet(null, 'infoCap'), client.intlGet(interaction.guildId, 'commandsVoiceJoin',
-                        { name: voiceChannel.name, id: voiceChannel.id, guild: voiceChannel.guild.name }));
+                        {   name: voiceChannel.name ? voiceChannel.name : client.intlGet(interaction.guildId, 'unknownVoiceChannelId'), 
```

Comment:
I dont think this will work either, this will give error when it tries to access the .name property during the condition. better to do the condition like this:

name: voiceChannel && voiceChannel.name ? voiceChannel.name : client.intl.....

Reply:
just changed it, thought it would fail if no name/id/guild is given and send the unknown phrase